demo/index.js

```diff
@@ -31,7 +33,21 @@ const callback = () => {
       },
       onDialNumber: (data, rawEvent) => {
         appendMsg(data, rawEvent);
-        cti.outgoingCall();
+        const { phoneNumber } = data;
+        state["phoneNumber"] = phoneNumber;
+        window.setTimeout(
+          () =>
+            cti.outgoingCall({
+              createEngagement: true,
```

Comment:
Just to clarify, does `createEngagement: true` tell HubSpot to create the engagement? So if they exclude this, we don't create the engagement.

Reply:
yes.  It helps maintain backward compatibility.  There is no major version update with this change.